Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -357,17 +357,22 @@ import BraintreeDataCollector
         return application.canOpenURL(paypalURL)
     }
 
-    private func launchPayPalApp(with payPalAppRedirectURL: URL, completion: @escaping (BTPayPalAccountNonce?, Error?) -> Void) {
+    private func launchPayPalApp(with payPalAppRedirectURL: URL, baToken: String?, completion: @escaping (BTPayPalAccountNonce?, Error?) -> Void) {
         apiClient.sendAnalyticsEvent(
             BTPayPalAnalytics.appSwitchStarted,
             linkType: linkType,
             payPalContextID: payPalContextID,
             payPalInstalled: payPalAppInstalled
         )
 
+        guard let baToken else {
```

Comment:
Take it or leave it - making `baToken` non-optional in this method and throwing this error before `launchPayPalApp()` is called

Reply:
✅ : https://github.com/braintree/braintree_ios/pull/1274/commits/225f6226f797b0eeb238e06c3d2c90ed71840d5d